src/meta-srv/src/keys.rs

```diff
@@ -228,6 +228,19 @@ pub struct StatValue {
     pub stats: Vec<Stat>,
 }
 
+impl StatValue {
+    /// Get the region number from stat value.
+    pub fn region_num(&self) -> Option<u64> {
+        for stat in self.stats.iter() {
+            match stat.region_num {
+                Some(region_num) => return Some(region_num),
+                None => continue,
+            }
+        }
+        None
```

Comment:
Can it be written like: `self.stats.iter().filter_map(|x| x.region_num).first()` ?

Reply:
It seems that there is no `fisrt` method.